pkg/api/api.go

```diff
@@ -180,10 +163,22 @@ func (a API) ReceiveEvent(w http.ResponseWriter, r *http.Request) {
 				return err
 			}
 
+			ts := time.Now()
 			if evt.Timestamp == 0 {
-				evt.Timestamp = time.Now().UnixMilli()
+				evt.Timestamp = ts.UnixMilli()
 			}
 
+			ctx, span := telemetry.UserTracer().Provider().
+				Tracer(consts.OtelScopeEvent).
```

Comment:
moved the event span down here as discussed

Reply:
With this now the root of the initial trace:

- Should we add a span event to this span that is the event's payload?

- Does invoke need one of these spans above the run?
  
  Currently it's Run -> Invoke Step -> Run
  Should it be Run -> Invoke Step -> Event -> Run?

Reply:
> Should we add a span event to this span that is the event's payload?
yes, it's added on the cloud side. but can be added for consistency.

> Does invoke need one of these spans above the run?
Keep the event span, but embed the event data into function run span as discussed.

Reply:
Change storing events into `Schedule`.

Reply:
Changing to store the Event's reference instead in separate discussion.

Reply:
Done at [ba2cdc7](https://github.com/inngest/inngest/pull/1248/commits/ba2cdc7a0c688addd79461cd2f9ad2e80d533862)